Userland/Libraries/LibJS/Tests/builtins/Temporal/PlainTime/PlainTime.from.js

```diff
@@ -64,4 +64,26 @@ describe("errors", () => {
             "Invalid time string '2021-07-06T23:42:01Z': must not contain a UTC designator"
         );
     });
+
+    test("ambiguous string must contain a time designator", () => {
```

Comment:
Might want to add a similar test with the same values but with the `T` designator showing they parse correctly?

Reply:
Line 86 :^)

> ``Temporal.PlainTime.from(`T${value}`);``